src/components/item-view/data-section/DataBreakdown.js

```diff
@@ -15,48 +21,92 @@ const DataBreakdown = ({
   ghg,
-  windowDimensions,
-  minRatio,
   item,
 }) => {
-  const { width, min } = windowDimensions;
-
   const text = {
     [ghg]: {
-      title: 'Carbon Emissions',
-      body:
-        `${foodData[item].raw.ghg}`,
+      title: 'Carbon',
+      rawValue:
+        `${foodData[item].raw.ghg} kgCO2`,
+      description: 'Write some stuff about how carbon emissions are created during food production',
     },
     [land]: {
-      title: 'Land Usage',
-      body:
-        `${foodData[item].raw.land}`,
+      title: 'Land',
+      rawValue:
+        `${foodData[item].raw.land} m2`,
+      description: 'Food uses lots of land. Land land land land, things about land.',
     },
     [water]: {
-      title: 'Water Usage',
-      body:
-        `${foodData[item].raw.water}`,
+      title: 'Water',
+      rawValue:
+        `${foodData[item].raw.water} l/kg`,
+      description: 'Plants get thirsty, and so do animals. To fix this problem we use water. Lots of it.',
     },
     [eutro]: {
       title: 'Eutrophication',
-      body:
-        `${foodData[item].raw.eutro}`,
+      rawValue:
+        `${foodData[item].raw.eutro} gPO₄eq`,
+      description: 'Oooooo Big Word. It refers to the amount of waste the food emitts that ends up as water pollution.',
     },
   };
 
   const body = [ghg, land, water, eutro].map((type) => (
-    <Grid item sm={12} className={styles.breakdownItem}>
-      <div className={styles.impactText}>
-        <Typography variant="subtitle1">{text[type].title}</Typography>
-        <Typography variant="body2" className={styles.body}>
-          {(width - min) / width >= minRatio.body ? text[type].body : null}
-        </Typography>
-      </div>
+    <Grid
+      item
+      xs={12}
+      className={styles.breakdownItem}
+    >
+      <Card
+        className={styles.impactText}
+      >
+        <CardContent>
+          <Grid
+            container
+            justify='space-evenly'
+          >
+            <Grid
+              item
+              xs={7}
+            >
+              <Typography
+                variant="h4"
+              >
+                {text[type].title}
+              </Typography>
+            </Grid>
+            <Grid
+              item
+              xs={5}
+            >
+              <Typography
+                variant="h5"
+                className={styles.rawValue}
+              >
+                {text[type].rawValue}
+              </Typography>
+            </Grid>
+            <Grid
+              item
+              xs={12}
+            >
+              <Typography
+                variant="body2"
+              >
+                {text[type].description}
+              </Typography>
+            </Grid>
+          </Grid>
+        </CardContent>
+      </Card>
     </Grid>
   ));
 
   return (
-    <Grid container spacing={1} className={styles.breakdownBox}>
+    <Grid
+      container
+      spacing={2}
+      className={styles.breakdownBox}
+    >
       {body}
     </Grid>
```

Comment:
This gets cut off on dimensions for my laptop screen 

![image](https://user-images.githubusercontent.com/27874045/88844941-6c067300-d1b1-11ea-8cc6-3e6f9e7daa12.png)

Reply:
Got it, that's fixed now, you'll just have to scroll through the modal on narrow screens